packages/js/data/src/crud/reducer.ts

```diff
@@ -48,6 +51,17 @@ export const createReducer = () => {
 							) ]: payload.error,
 						},
 					};
+				case TYPES.GET_ITEMS_TOTAL_COUNT_SUCCESS:
+					return {
+						...state,
+						itemsCount: {
+							...state.itemsCount,
+							[ getTotalOrderCountResourceName(
```

Comment:
It makes no sense to me why a general function is calling something that specifically references "Order". Either I'm misinterpreting things or `getTotalOrderCountResourceName` should be renamed.

Reply:
Aaah yes, I missed that from my copy paste (facepalm)